packages/scrabble-solver/src/i18n/fa.json

```diff
@@ -0,0 +1,56 @@
+{
+  "cell.filter-cell": "مقصد - کلیک برای تغییر",
+  "cell.set-blank": "علامت گذاری به عنوان خالی",
+  "cell.set-not-blank": "علامت گذاری به عنوان غیر خالی",
+  "cell.toggle-direction": "جهت تایپ - کلیک برای تغییر",
+  "common.blanks": "خالی",
+  "common.clear": "پاک کردن",
+  "common.close": "بستن",
+  "common.consonants": "حروف صامت",
+  "common.loading": "در حال بارگزاری",
```

Comment:
The use of spaces here causes an issue with how this text is displayed:

![image](https://user-images.githubusercontent.com/6830683/212016280-e9a34cb4-f855-4871-b805-c1ed8261bef0.png)

Reply:
This patch fixes it:
```patch
diff --git a/packages/scrabble-solver/src/components/Loading/Loading.tsx b/packages/scrabble-solver/src/components/Loading/Loading.tsx
index b76e52c3..903a4ecd 100644
--- a/packages/scrabble-solver/src/components/Loading/Loading.tsx
+++ b/packages/scrabble-solver/src/components/Loading/Loading.tsx
@@ -13,9 +13,15 @@ interface Props {
   wave?: boolean;
 }
 
+const prepareContent = (message: string): string[][] => {
+  const uppercased = message.toLocaleUpperCase();
+  const parts = uppercased.split(' ');
+  return [parts];
+};
+
 const Loading: FunctionComponent<Props> = ({ className, wave = true }) => {
   const translate = useTranslate();
-  const content = useMemo<string[][]>(() => [[translate('common.loading').toUpperCase()]], [translate]);
+  const content = useMemo(() => prepareContent(translate('common.loading')), [translate]);
 
   return (
     <div className={classNames(styles.loading, className)}>
```

![image](https://user-images.githubusercontent.com/6830683/212016224-d9255e16-0e4b-43fd-a615-f51a1e544113.png)

Reply:
hmmm, im not very familiar with PR workflow, and i supposed to make this changes and made a new commit or these are already done?

Reply:
This needs to be a new commit.
To apply a patch, copy all the code I pasted above, save it as a new file in the repository, call it e.g. `patch.diff`, and then apply the file as a patch: `git apply patch.diff`. Then you can delete the `patch.diff` file and commit the changes.

Reply:
cool, learning new stuff :)
i followed the steps, but I'm getting an error ->  "error: corrupt patch at line 21" if I had to guess maybe its a windows problem.

Reply:
Fixed: adafdef1